spec/solver_spec.rb

```diff
@@ -20,7 +22,7 @@
 
   describe '#fizzbuzz' do
     it 'returns constructed string' do
-      expect(@solver.fizzbuzz(15)).to eq '1, 2, fizz, 4, buzz, fizz, 7, 8, fizz, buzz, 11, fizz, 13, 14, fizzbuzz'
+      expect(@solver.fizzbuzz(15)).to eq 'fizzbuzz'
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/keebot-png/TDD-project/pull/1#discussion_r1102551406 please update and add the below test cases: 
 ```
   expect(resolver.fizzbuzz(3)).to eql('fizz')  # N is divisible by 3
    expect(resolver.fizzbuzz(5)).to eql('buzz')  # N is divisible by 5
    expect(resolver.fizzbuzz(7)).to eql('7')  # N is not divisible by 3 or 5
```